tests/manual-test-cases/Group18-VIC-UI/18-2-VIC-UI-Uninstaller.robot

```diff
@@ -22,10 +22,14 @@ Suite Teardown  Uninstall VCH
 
 *** Test Cases ***
 Ensure Vicui Is Installed Before Testing
```

Comment:
This should really be in the suite setup, and this whole structure is pretty error prone - if any of these attempted uninstalls suddenly work for some reason then all the rest of the tests fail.  Better to have a test setup that checks whether it is installed first, then installs if necessary on each test.